release/RELEASE_PROCESS_v2.md

```diff
@@ -77,17 +77,17 @@ You can repeat same steps for RC2 or other release candidates if needed
 #### Updating the version in master 
 This will be the last commit before the release and last one to be cherry-picked into release branch. So, we have to update the release version in master to reflect the latest release we are at.  
 1. Create a PR with the following changes to update the version number in the following places in the master:
-   1. [VERSION](../python/VERSION) to `${MAJOR}.${MINOR}.${PATCH}-${RELEASE_VERSION}`
-   2. [quick_install.sh](../hack/quick_install.sh#L36) to `v${MAJOR}.${MINOR}.${PATCH}-rc${RELEASE_VERSION}`
-   3. [Chart.yaml in kserve-crd](../charts/kserve-crd/Chart.yaml#L3) to `v${MAJOR}.${MINOR}.${PATCH}-rc${RELEASE_VERSION}`
-   4. [Chart.yaml in kserve-resources](../charts/kserve-resources/Chart.yaml#L3) to `v${MAJOR}.${MINOR}.${PATCH}-rc${RELEASE_VERSION}`
-   5. [values.yaml in kserve-resources](../charts/kserve-resources/values.yaml#L2) to `v${MAJOR}.${MINOR}.${PATCH}-rc${RELEASE_VERSION}`
-2. Add a new version `v${MAJOR}.${MINOR}.${PATCH}-rc${RELEASE_VERSION}` in the `RELEASES` array in [generate-install.sh](../hack/generate-install.sh). Example: Refer [this commit](https://github.com/rachitchauhan43/kserve/commit/6e9bd24ea137a3619da3297b4ff000379f7b2b38#diff-5f8f3e3a8ca601067664c7bf00c05aa2290a6ba625312754856ec873b840b6dbR42)
+   1. [VERSION](../python/VERSION) to `${MAJOR}.${MINOR}.${PATCH}`
+   2. [quick_install.sh](../hack/quick_install.sh#L36) to `v${MAJOR}.${MINOR}.${PATCH}`
+   3. [Chart.yaml in kserve-crd](../charts/kserve-crd/Chart.yaml#L3) to `v${MAJOR}.${MINOR}.${PATCH}`
+   4. [Chart.yaml in kserve-resources](../charts/kserve-resources/Chart.yaml#L3) to `v${MAJOR}.${MINOR}.${PATCH}`
+   5. [values.yaml in kserve-resources](../charts/kserve-resources/values.yaml#L2) to `v${MAJOR}.${MINOR}.${PATCH}`
```

Comment:
Correct me if I am wrong but I think in this section, we are releasing the GA version, not the release candidate any more.